### ISSUE TITLE
deegree maven plugin does clean up deegree-compliance-tests directory

### ISSUE THREAD

**Issue by tfr42** [deegree/deegree3#1072]
This leads to growing size of the deegree-compliance-tests web app.

Workaround is available here: https://github.com/deegree/deegree3/compare/master...lat-lon:cleanPluginComplianceTests

**Comment by lgoltz** [comment]
Testfailures are caused by a change in https://github.com/deegree/deegree3/pull/1058/files#diff-41768028a4f464394c40f29346f004f1121204b40501c7e168d6ee04fa592d98
Adds: `graph.removeNode( id )` 
This removes nodes from the graph while other (dependents of the resource to destroy) are accessed.

First try: 
 1. Collecting all resources to remove
 2. Remove collected resources

This leads to a NPE in org.deegree.workspace.SimpleSqlFeatureStoreTest#testDestroyInitializeSingle():

```
org.deegree.workspace.ResourceInitException: Unable to build resource FeatureStoreProvider:simplesql-ok: null

	at org.deegree.workspace.standard.DefaultWorkspace.init(DefaultWorkspace.java:398)
	at org.deegree.workspace.SimpleSqlFeatureStoreTest.testDestroyInitializeSingle(SimpleSqlFeatureStoreTest.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
...
Caused by: java.lang.NullPointerException
	at org.deegree.feature.persistence.simplesql.DbFeatureUtils.determineFeatureType(DbFeatureUtils.java:110)
	at org.deegree.feature.persistence.simplesql.SimpleSQLFeatureStore.init(SimpleSQLFeatureStore.java:419)
	at org.deegree.workspace.standard.DefaultWorkspace.init(DefaultWorkspace.java:388)
	... 25 more
```

It seems that the the dependent JDBCConnection is not Initialised as expected.

**Comment by stephanr** [comment]
There is a Jenkins plugin that has a more general approach to cleaning up the entire worksapce folder.

Plugin https://plugins.jenkins.io/ws-cleanup/

But this requires to ad the following to the Jenkinsfile
```
post {
  cleanup {
    cleanWs()
  }
}
```

A detailed configuration for the Pipeline can be generated through the Jenkins UI at http://buildserver.deegree.org/job/deegree-3.4-pipeline/pipeline-syntax/

**Comment by tfr42** [comment]
Fixed by adding post action: https://github.com/deegree/deegree3/commit/5b6812a47e9daac228d98c2aa98998cd4f8b9995

**Comment by tfr42** [comment]
Root cause also resolved by replacing the deegree-maven-plugin (#1169).
